integration_tests/src/main/python/conditionals_test.py

```diff
@@ -47,13 +49,20 @@ def test_case_when(data_gen):
     gen = StructGen(gen_cols, nullable=False)
     command = f.when(f.col('_b0'), f.col('_c0'))
     for x in range(1, num_cmps):
-        command = command.when(f.col('_b'+ str(x)), f.col('_c' + str(x)))
-    command = command.otherwise(s1)
+        command = command.when(f.col('_b'+ str(x)), f.col('_c' + str(x)))\
+                         .when(f.lit(True), f.col('_c' + str(x)))
```

Comment:
My concern with putting a lit true for here and below on line 54 is that it ensures that the evaluation essentially stops at that point, and it does not matter what we evaluate after this.  Internally we are walking from the bottom up, but the test should be agnostic of that.

I would rather we have them be below. as separate case/when statements

Reply:
Done.